frontend/app/src/components/home/Home.svelte

```diff
@@ -652,6 +652,10 @@
                     kind: "show_pinned",
                 },
             ]);
+            let chat = $selectedChatStore!;
```

Comment:
feels like this should probably be done inside the PinnedMessages.svelte component where the pinned messages are _actually_ loaded.